src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -491,27 +491,43 @@ internal Uri CreateIssuePreview(string projectName, string teamName, IReadOnlyDi
         }
 
         /// <summary>
-        /// Retains characters within ascii range below 127 and returns
-        /// an escaped representation of the string
+        /// Substitutions for characters when creating the bug URL
+        /// </summary>
+        internal static IReadOnlyDictionary<char, char> CharacterSubstitutions = new Dictionary<char, char>()
+        {
+            { '\u00b7', '-' }, // middle dot (183) to dash (45)
+        };
+
+        /// <summary>
+        /// replaces characters if substitution defined and
+        /// returns an escaped representation of the string
         /// </summary>
         /// <param name="str"></param>
         /// <returns></returns>
         internal static string EscapeForUrl(string str)
         {
+            // characters such as the middle dot can 
+            // cause issues during navigation, so we only allow
+            // a basic set of characters when building the URL
             var sb = new StringBuilder();
             foreach (var c in str)
             {
-                // characters such as the middle dot (ascii 183) can 
-                // cause issues during navigation, so we only include
-                // a basic set of characters when building the URL
-                if (c <= 127)
+                if (CharacterSubstitutions.TryGetValue(c, out char replaced))
+                {
+                    sb.Append(replaced);
+                }
+                else if (c <= 127)
                 {
                     sb.Append(c);
                 }
+                else
+                {
+                    sb.Append('?');
```

Comment:
I feel like we might want something other than a common character (?) here to make it clear that we modified the string. I would suggest �, but that won't work for obvious reasons...maybe some kind of enclosed ? like <?> ?

Reply:
I was hoping to keep the replacement a single character; if you feel strongly about this I'm happy to update. IMHO the ? seems sufficient for the case we're trying to handle - lmk